apps/storefront/src/pages/AccountSetting/index.tsx

```diff
@@ -237,10 +223,32 @@ function AccountSetting() {
       try {
         const isValid = await validateEmailValue(data.email);
 
+        if (!isValid) {
+          setError('email', {
+            type: 'custom',
+            message: b3Lang('accountSettings.notification.emailExists'),
+          });
+        }
+
         const emailFlag = emailValidation(data);
 
+        if (!emailFlag) {
```

Comment:
these `flag` names aren't very descriptive, what do you think to turning them into something like

```tsx
if (isValidEmail(data.email)) { 
    // etc
```

in another commit/PR/ticket?

Reply:
can do